src/entry_table.rs

```diff
@@ -179,21 +179,25 @@ impl<const UID_LENGTH: usize, const KWI_LENGTH: usize> EntryTableValue<UID_LENGT
     ///
     /// - `k_value`     : `𝐾_value`
```

Comment:
Non unicode character '𝐾' in comment

Reply:
Yes, I originally used a copy/paste from the LaTeX PDF, thus this fancy K. Is it a bad thing ?

Reply:
Non-unicode characters may be undefined / different on other os / architectures.
This is a comment and I'm pretty sure rust compiler wouldn't mind it, but still, in a concern of portability I would avoid those as much as possible
```suggestion
    /// - `k_value`     : `K_value`
```